### PR TITLE
Move metadata items across Steward Organisations bulk action

### PR DESCRIPTION
**Pull request by s-i-l-k-e**
Fixes issues: #

Sanity checks:
- [ ] Have tests been run locally?
- [ ] Does this change any javascript? If so, have you checked that the UI works as expected?
- [ ] If there are data model changes, are relevant data standards referenced - in code?
- [ ] If there are deviations from any specified standards, are the reasons for changes documented?
- [ ] If there are UI changes have you documented whats changed so the help documentation can be updated?

### CODE REVIEW

File: python/aristotle-metadata-registry/aristotle_mdr/contrib/search_backends/facetted_whoosh.py [hunk 1]
```diff
@@ -70,19 +70,6 @@ def get_writer(self, index):
         return WriterWithFasterSpellingUpdate(
             self.storage, self.get_spell_checker(), self.spelling_fields, index
         )
```
Comment (LegoStormtroopr) [inline python/aristotle-metadata-registry/aristotle_mdr/contrib/search_backends/facetted_whoosh.py]: Why are we removing this function?

File: python/aristotle-metadata-registry/aristotle_mdr/models.py [hunk 2]
```diff
@@ -1869,6 +1869,15 @@ def is_stewardship_organisation_admin(self, org=None):
...
         return StewardOrganisationMembership.objects.filter(**kwargs).exists()
 
+    @property
+    def stewardship_organisations(self):
+        """The list of Stewardship Organisations the user is a member in, or all, if they are a superuser """
+        if self.user.is_superuser:
+            return StewardOrganisation.objects.all()
+        else:
+            # They are not a superuser
+            return StewardOrganisation.objects.visible(self.user).filter(members__user=self.user)
```
Comment (LegoStormtroopr) [inline python/aristotle-metadata-registry/aristotle_mdr/models.py]: I think we have a  function in groups for this

Comment (s-i-l-k-e) [reply python/aristotle-metadata-registry/aristotle_mdr/models.py]: Thanks for spotting that. I'll have a look for the function.

File: python/aristotle-metadata-registry/aristotle_mdr/contrib/search_backends/facetted_whoosh.py (hunk 1 above)
Comment (s-i-l-k-e) [reply python/aristotle-metadata-registry/aristotle_mdr/contrib/search_backends/facetted_whoosh.py]: It was commented out in 2016, happy to look into keeping it though.

**Pull request merged**